util/java/libtiled-java/README.MD

```diff
@@ -1,5 +1,5 @@
 ---------------
- libtiled Java
+libtiled Java
```

Comment:
This file was renamed to have a Markdown extension, but its formatting does not generally appear to be updated to actual Markdown. I'd suggest to either not rename or to finish the formatting.

Reply:
@bjorn I will rename extensions and will remove .gitignore from the `\tiled\util\java` folder.

Reply:
@bjorn I've updated extensions and removed .gitignore from the `tiled\util\java` folder.

.gitignore files in other folder are necessary. Because every folder represents different project.

`pom.xml` file in the `tiled\util\java` allows to build both projects from the terminal (command line) via one command.

Reply:
@bjorn did not notice comments regarding markdown format. Will check that and update soon.

Reply:
@bjorn all checks were passed.